akka-sample-camel-java/build.sbt

```diff
@@ -2,6 +2,7 @@ organization := "com.typesafe.akka.samples"
 name := "akka-sample-camel-java"
 
 scalaVersion := "2.12.6"
+crossScalaVersions := Seq("2.12.6", "2.11.11")
```

Comment:
I don't think we should complicate the build files of the samples by adding crossScalaVersions. That's an advanced feature that libraries but not applications may use. We want to promote latest version.

Reply:
The reason of adding crossScalaVersions is that build files without crossScalaVersions will throw:
```scala
java.lang.RuntimeException: Switch failed: no subprojects list "2.11.11" (or compatible version) in crossScalaVersions setting.
```
when executing:
```bash
sbt ++\$TRAVIS_SCALA_VERSION test
```

Or we follow the suggestion from sbt - If you want to force it regardless, call ++ 2.11.11!.

Reply:
but shouldn't we remove Scala 2.11 from the Travis build instead?

Reply:
Got it. I will push soon. :)